polynote-frontend/polynote/interpreter/client_interpreter.ts

```diff
@@ -123,6 +129,9 @@ export class ClientInterpreter {
                     this.receiver.inject(new CellResult(id, res))
                 }
             })
+        }).catch(() => {}).finally(() => {
```

Comment:
Does this `catch` swallow any errors? Might we want to print them out to the console?

Reply:
The only thing caught here is that the cell we were waiting on was cancelled.

I guess this is "exception as flow control" which isn't great (it's only using `reject` to bypass the part where it interprets). I'll change it to pass a boolean `cancelled` instead.